File: java/Gate Two/Level4Methods.java

public class Level4Methods{

	public static int addition(int numberOne, int numberTwo){
		return numberOne + numberTwo;
	}
	
	public static String evenCheck(int number){
		if(number % 2 == 0){
			return "Is Even";
		}
		else{
			return "Is Not Even";
		}
	}

	public static int squared (int number){
		return number * number;
	}

	public static double celsiusToFarenheit(int temperature){
		return temperature * 9.0/5.0 + 32.0;
	}
	
	public static boolean isPrime (int number){
		boolean prime = true;
		for (int numbers = 2; numbers < number / 2; numbers++){
			if (number % numbers == 0){
				prime = false;
			}
		}
		return prime;
	}
	
	public static int largestOfThree (int firstNumber, int secondNumber, int thirdNumber){
		int[] listOfNumbers = {firstNumber, secondNumber, thirdNumber};
		int largest = 0;
		for (int numbers: listOfNumbers){
			if (numbers > largest){
				largest = numbers;
			}
		}
		return largest;
	}

	public static double simpleIntrest (int principal, int intrestRate, int time){
		return principal * intrestRate * time * 0.01;
	}
	

	public static void multiplicationTable(int number){
		for(int index = 0; index < 11; index++){
			System.out.printf("%d x %d = %d \n", number, index, (number * index));
		}
	}

	public static int areaOfRectangle(int length, int width){
		return length * width;
	}

	public static int reverseNumber(int number){

		int reversedNumber = 0;
		while(number != 0){
			int lastDigit = number % 10;
			number /= 10;
			reversedNumber *= 10;
			reversedNumber += lastDigit;
		}
		return  reversedNumber;

	}
} 

File: javaScript/Gate Two/Level4Methods.js
function addition(number_one, number_two){
    return number_one + number_two;}
 
function even_check(number){
	
    if (number % 2 == 0){
        return "Is Even";}
    else{
        return "Is not Even";}
	}

function squared(number){
    return number ** 2;}

function celsius_to_farenheit(temperature){
    return temperature * 9/5 + 32;}

function is_prime(number){
    let prime = true;

    for (let numbers = 2; numbers < number/2; numbers ++){
        if (number % numbers == 0){
            prime = false;}
	}
    return prime;}

function largest_of_three(first_number, second_number, third_number){
    list_of_numbers = [first_number, second_number, third_number];
    largest = 0;
    for (let numbers of list_of_numbers){
        if (numbers > largest){
            largest = numbers;}
	}
    return largest;}
    
function simple_intrest(principal, intrest_rate, time){
    return principal * intrest_rate * 0.01 * time;}

function multiplication_table(number){
    list_of_table = [];
    for (let numbers = 0; numbers <= 10; numbers++){
        list_of_table += [`${number} x ${numbers} = ${number * numbers}`];}
    return list_of_table;}

function area_of_rectangle(length, width){
    return length * width;}

function reverse_number(number){
    let reversed_int = 0
    while (number != 0){
        let last_digit = number % 10;
        number /= 10;
        reversed_int = reversed_int * 10 + last_digit;}
    return reversed_int;}
        





        
